### DISCUSSION TITLE
When flet is able to support c#?

### DISCUSSION THREAD

**Discussion by i-py-cs** [flet-dev/flet#3812] [Ideas]
As title

**Comment by baseplate-admin** [comment]
I think current priority is python only.

Porting a framework to another language is a lot of effort. Flet community is currently focusing on python

**Reply by matthewblott** [reply]
Maybe it shouldn't be on the homepage that other languages are coming :-)

**Reply by baseplate-admin** [reply]
> Maybe it shouldn't be on the homepage that other languages are coming :-)

It is on the roadmap tbh, the creator of this project wanted flet to be language agnostic. But ETA for the implementation is a bit unknown.

Just look at python implementation. It took a lot lot of work to get one implementation working properly, other implementation might be just as time consuming. After the python port is complete the creator will move to other language binding.

**Reply by FeodorFitsner** [reply]
You guys are both right: we should move to another language once Python is polished and it shouldn't be on the home page 😅

**Reply by Muddassir-Farooq-official** [reply]
> You guys are both right: we should move to another language once Python is polished and it shouldn't be on the home page 😅

It is a humble request kindly add notifications system in flet , is there any chance in future that flet become an independent platform and don't depend on flutter
